.cicd/eosio-contracts-unittest.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+
+[[ ! -z "$CONTRACTS_VERSION" ]] || export CONTRACTS_VERSION="$(cat "$PIPELINE_CONFIG" | jq -r '.dependencies["eosio.contracts"]')"
+git clone -b "$CONTRACTS_VERSION" https://github.com/EOSIO/eosio.contracts.git 
+
+# Docker Commands
+ARGS=${ARGS:-"--rm --init -v $(pwd):$MOUNTED_DIR"}
+. $HELPERS_DIR/docker-hash.sh
+
+if [[ $BUILDKITE == true ]]; then
+    # Generate Base Images
+    $CICD_DIR/generate-base-images.sh
```

Comment:
We should not call the `generate-base-images.sh` script in the test steps.

This script creates docker base-images using the source code in the `.cicd/platforms` folder. However, here, this script will be a no-op because if these platforms files have changed, the build steps will already have built the new base-images by now and docker will pull them down for us.

Reply:
Done.